source/OutfitSale.cpp

```diff
@@ -0,0 +1,101 @@
+/* OutfitSale.cpp
+Copyright (c) 2021 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "DataNode.h"
+#include "OutfitSale.h"
+#include "Set.h"
+#include "Sold.h"
+
+#include <string>
+
+
+
+void OutfitSale::Load(const DataNode &node, const Set<Outfit> &items)
+{
+	for(const DataNode &child : node)
+	{
+		const std::string &token = child.Token(0);
+		bool remove = (token == "clear" || token == "remove");
+		if(remove && child.Size() == 1)
+			this->clear();
+		else if(remove && child.Size() >= 2)
+			this->erase(items.Get(child.Token(1)));
+		else if(token == "add" && child.Size() >= 2)
+			(*this)[items.Get(child.Token(1))].SetBase(child.Size() > 2 ? 
+				child.Value(2) : 0., child.Size() > 3 ? child.Token(3) : "");
+		else if(token == "hidden" || token == "import")
```

Comment:
by not likely you mean impossible
more seriously what kind of syntax do you propose ? repeating the import/outfit for all outfits ? (that already works I just find "breaking of syntax" to be way more elegant and simpler to use/understand)
because the alternative would cause the issue that you have to give a price when using import/hidden

Reply:
also the add and import is without "" meaning it is a different syntax.